packages/scaffold/src/views/w3m-account-view/index.ts

```diff
@@ -162,6 +155,25 @@ export class W3mAccountView extends LitElement {
   }
 
   // -- Private ------------------------------------------- //
+  private onrampTemplate() {
+    const { enabled } = OnRampController.state
```

Comment:
Maybe new lines between variables and return statements? 
```suggestion
    const { enabled } = OnRampController.state
    
```